From cff6214449dc4477deb071b0cd23105c0cac3704 Mon Sep 17 00:00:00 2001
From: Richard Gong <richard.gong@intel.com>
Date: Mon, 12 Apr 2021 13:07:51 -0500
Subject: [PATCH 25/41] Revert "HSD #16013024105: fpga: stratix10-soc: stop
 kernel thread"

commit 9c66d476bb9296ff442c94057f41bd555727d4f5 from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.114-lts

This reverts commit c7bfe97dd00781aee819c4cb0a75bed549f9037f.

Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 drivers/fpga/stratix10-soc.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/fpga/stratix10-soc.c b/drivers/fpga/stratix10-soc.c
index 6ba59841fbf7..9ab7afd28a64 100644
--- a/drivers/fpga/stratix10-soc.c
+++ b/drivers/fpga/stratix10-soc.c
@@ -486,12 +486,10 @@ static int s10_probe(struct platform_device *pdev)
 			       NULL, 0, s10_fw_version_callback);
 	if (ret) {
 		dev_err(dev, "couldn't get firmware version\n");
-		stratix10_svc_done(priv->chan);
 		fpga_mgr_free(mgr);
 		goto probe_err;
 	}
 
-	stratix10_svc_done(priv->chan);
 	platform_set_drvdata(pdev, mgr);
 	return ret;
 
-- 
2.18.1

